Update README.md

Spone opened the pull request:
small mistake fixed here to ;)

Spone committed:
Update README.md

small mistake fixed here to ;)

darsain commented:
Hm... actually, panning should have only play/pause/toggle, and sprite should have all the options. Can you also change that? :)

Spone committed:
Update README.md

moved the available methods from panning to sprite, as [requested](https://github.com/darsain/motio/pull/20#issuecomment-62592421)

Spone commented:
Done c872eda
